app/src/lib/app-state.ts

```diff
@@ -170,6 +170,9 @@ export interface IAppState {
   /** The width of the files list in the stash view */
   readonly stashedFilesWidth: IConstrainedValue
 
+  /** The width of the files list in the pull request files changed view */
+  readonly pullRequestFilesListWidth: IConstrainedValue
```

Comment:
I debated whether it made sense to keep track of this at the app level or since it is not used for anything outside that put it on the dialog state or pull request state. Reason I chose to keep it here was just for consistency of where we keep all the other file width tracking logic and that we seem to have a convention of accessing local storage here and then adding it to the app state. Unless we move accessing/updating local storage down to the dialog, we would still have the same about of update methods in the dispatcher/app-store. Completely open to moving this if anyone feels it makes more sense to scope it down to the dialog.